pkg/discovery/generator.go

```diff
@@ -8,7 +8,10 @@ import (
 
 // Generate takes care of IP generation
 func Generate(pattern string) []string {
-	re, _ := regexp.Compile(`\[(.+):(.+)\]`)
+	re, err := regexp.Compile(`\[(.+):(.+)\]`)
```

Comment:
@cpuguy83  Can you explain explicitly? I'm afraid I did't get your point of package init :(